core/src/i18n/en.yaml

```
@@ -117,7 +119,67 @@ discovery-and-network:
 functions:
   title: Functions
   variable:
-    tooltip: This variable comes from a Resource. To get the value, follow the link.
+    buttons:
+      add: Add environment variable
+      open:
+        custom: Custom Variable
+        secret: Secret Variable
+        config-map: Config Map Variable
+    create-modal:
+      title:
+        custom: Create Custom Variable
+        secret: Create Secret Variable
+        config-map: Create Config Map Variable
+    edit-modal:
+      title:
+        custom: Edit Custom Variable
+        secret: Edit Secret Variable
+        config-map: Edit Config Map Variable
+    errors:
+      empty: Variable is empty.
+      duplicated: Duplicated variable name.
+      invalid: Variable name must contain alphanumeric characters, can contain '_', no blank spaces.
+      invalid-config: Select the config map name and key
```

```suggestion
      invalid-config: Select the ConfigMap name and key.
```

The other error msgs all have full stops, so I'd add them here too. It might be worth discussing in the team whether we want full stops with error msgs or not, for the sake of consistency.